wiki/Beatmap_Editor/Distance_Snap/en.md

```diff
@@ -33,3 +42,5 @@ When setting the distance, you should focus on certain points having about the s
 ## Using Distance Snap
 
 Distance Snap is just a **tool** to maintain proportional distance between two hit-objects, a "crutch" if you will. It works well in most situations but over-reliance will make your difficulty predictable and with limited working space. While it is not wrong to use Distance Snap, remember that it is just a tool intended to suit the mapper's purpose. A change in distance should make sense and follow to the rhythm well. Distance Snap is just a helper and does not constitute a professional advice, that would be in the [beatmap forum](https://osu.ppy.sh/community/forums/56) if you need one.
```

Comment:
```suggestion
**Distance snap** is simply a *tool* in the [Beatmap editor](/wiki/Beatmap_editor) used to maintain proportional distance between two [hit objects](/wiki/Hit_object) ( a "crutch" of sorts). The distance snap tool works well in most situations, but over-reliance on it can make a [beatmap](/wiki/Beatmaps]'s difficulty predictable and leave mappers with limited working space. While it is not wrong to use the distance snap tool, it is important to understand it is a tool intended to suit mapper's specific purposes.

*For mappers interested in "proffesional advice regarding spacing of hit objects, see the [beatmap forums](https://osu.ppy.sh/community/forums/56)

 A change in distance should make sense and follow the rhythm well. Distance snap is intended to *help* mappers and does not constitute as "professional" advice.
```
First-person POV is a no-no